certification/engine/engine.go

```diff
@@ -84,21 +97,9 @@ var nameToChecksMap = map[string]certification.Check{
 	hasMinimalVulnerabilitiesCheck.Name(): hasMinimalVulnerabilitiesCheck,
 	hasUniqueTagCheck.Name():              hasUniqueTagCheck,
 	hasNoProhibitedCheck.Name():           hasNoProhibitedCheck,
-	validateOperatorBundle.Name():         validateOperatorBundle,
 }
 
-var containerPolicyChecks = map[string]certification.Check{
-	runAsNonRootCheck.Name():              runAsNonRootCheck,
-	underLayerMaxCheck.Name():             underLayerMaxCheck,
-	hasRequiredLabelCheck.Name():          hasRequiredLabelCheck,
-	basedOnUbiCheck.Name():                basedOnUbiCheck,
-	hasLicenseCheck.Name():                hasLicenseCheck,
-	hasMinimalVulnerabilitiesCheck.Name(): hasMinimalVulnerabilitiesCheck,
-	hasUniqueTagCheck.Name():              hasUniqueTagCheck,
-	hasNoProhibitedCheck.Name():           hasNoProhibitedCheck,
-}
-
-var operatorPolicyChecks = map[string]certification.Check{
+var operatorChecks = map[string]certification.Check{
```

Comment:
The Policy name was specifically chosen, to represent the "group of Checks". It should remain.

Reply:
I am in favor of shorter names as much as possible. I was thinking of either `OperatorChecks` or `OperatorPolicy`. Not sure if we need both?

Reply:
I think I'd be ok with OperatorPolicy... Then makeCheckList(OperatorPolicy) would read pretty descriptively to me.

Reply:
@tonytcampbell Could you please verify if the terminology is consistent?

Reply:
Agreed - Policy should be used here. 

Here's @tonytcampbell's confirmation from previous discussion: https://github.com/redhat-openshift-ecosystem/openshift-preflight/pull/9#discussion_r664898636